Sources/PT.PM.JavaScriptParseTreeUst/JavaScriptEsprimaErrorHandler.cs

```diff
@@ -42,27 +56,18 @@ public void ThrowError(int index, int line, int column, string message)
             }
             else
             {
-                Logger.LogError(CreateException(index, message));
+                var exception = CreateException(index, message);
+                Logger.LogError(exception);
+                result = new ParserException(exception.ToString());
             }
 
             LastErrorIndex = index;
-        }
 
-        public void Tolerate(ParserException error)
-        {
-            if (Tolerant)
-            {
-                RecordError(error);
-            }
-            else
-            {
-                throw CreateException(error.Index, error.Message);
-            }
+            return result;
         }
 
         public void RecordError(ParserException error)
         {
-            Logger.LogError(CreateException(error.Index, error.Message));
```

Comment:
`RecordError` does nothing now? Is this intentional? Then why can't we remove it altogether?

Reply:
I removed it because of errors duplication in the log. They are handled in another method.

Reply:
But why keep the method?

Reply:
I think you should've added a comment stating why the method is left empty...